cmd/workflow.go

```diff
@@ -1,40 +1,174 @@
 package cmd
 
 import (
+	_ "embed"
+	"fmt"
+	"os"
+	"strings"
+
 	"github.com/spf13/cobra"
 
 	e "github.com/cloudposse/atmos/internal/exec"
 	"github.com/cloudposse/atmos/pkg/schema"
+	"github.com/cloudposse/atmos/pkg/ui/markdown"
 	u "github.com/cloudposse/atmos/pkg/utils"
 )
 
+//go:embed markdown/workflow.md
+var workflowMarkdown string
+
+// ErrorMessage represents a structured error message
+type ErrorMessage struct {
+	Title      string
+	Details    string
+	Suggestion string
+}
+
+// renderError renders an error message using the markdown renderer
+func renderError(msg ErrorMessage) error {
+	renderer, err := markdown.NewRenderer(
+		markdown.WithWidth(80),
```

Comment:
For docs, we have this configurable and dynamic.

Reply:
Seems like we shouldn't have this constant in every file.

Reply:
fixed